lighttpd/tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+import time
+
+from six.moves.urllib.request import urlopen
+from six.moves.urllib import error
+from copy import deepcopy
+
+from datadog_checks.dev import docker_run
+from datadog_checks.lighttpd import Lighttpd
+
+from . import common
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    with docker_run(common.COMPOSE_FILE):
+        attempts = 0
```

Comment:
This part should be made into a function, and passed to docker_run as `conditions`